fractal_server/app/schemas/v2/task_collection.py

```diff
@@ -77,32 +75,3 @@ def package_version_validator(cls, v, values):
                 "Cannot provide version when package is a Wheel file."
             )
         return v
-
-
-class TaskCollectStatusV2(BaseModel):
-    """
-    TaskCollectStatus class
-
-    Attributes:
-        status:
-        package:
-        venv_path:
-        task_list:
-        log:
-        info:
-    """
-
-    status: Literal["pending", "installing", "collecting", "fail", "OK"]
```

Comment:
Let's encode this in an `CollectionStatusV2` Enum, in `fractal_server.app.schemas.v2.task_collection`

Reply:
This is useful also in 
* `fractal_server/tasks/v2/background_operations.py`
* `fractal_server/tasks/v2/background_operations_ssh.py`